writer.go

```diff
@@ -58,3 +72,16 @@ func (w *writer) prefix(ev Event) string {
 		ev.Source().Name(),
 		ev.Source().Container())
 }
+
+func tryJsonPrettyPrint(o []byte) []byte {
+	t := &bytes.Buffer{}
+	err := json.Indent(t, o, "", "    ")
+	if err != nil {
+		return nil
+	}
```

Comment:
you can return [`t.Bytes()`](https://golang.org/pkg/bytes/?m=all#Buffer.Bytes) here; `ioutil.ReadAll(t)` isn't necessary.

Reply:
Done.